knowledge/science/botony/poison_ivy/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: Giovanna Miranda  
+domain: medical
+seed_examples:
+- answer: Poison ivy is a type of allergenic plant in the genus Toxicodendron native to Asia and North America. 
+  question: Where is Poison Ivy  native to?
+- answer: |They are well known for causing urushiol-induced contact dermatitis, an itchy, irritating, and sometimes painful rash, in most people who touch them. 
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!